src/website/views/components/data-table/_examples.pug

```diff
@@ -0,0 +1,5827 @@
+//-                   <span class="chi-pagination__label">240 results</span>
+//-                 </div>
+//-                 <div class="chi-pagination__page-size">
+//-                   <div class="chi-dropdown">
+//-                     <button class="chi-button -flat -px--1 chi-dropdown__trigger" id="pagesize-3">20</button>
+//-                     <div class="chi-dropdown__menu -w--xs">
+//-                       <a class="chi-dropdown__menu-item -active" href="#">20</a>
+//-                       <a class="chi-dropdown__menu-item" href="#">40</a>
+//-                       <a class="chi-dropdown__menu-item" href="#">60</a>
+//-                       <a class="chi-dropdown__menu-item" href="#">80</a>
+//-                       <a class="chi-dropdown__menu-item" href="#">All</a>
+//-                     </div>
+//-                   </div>
+//-                   <span class="chi-pagination__label">per page</span>
+//-                 </div>
+//-               </div>
+//-               <div class="chi-pagination__center">
+//-                 <div class="chi-pagination__button-group chi-button-group">
+//-                   <button class="chi-button -icon -flat" aria-label="Previous page">
+//-                     <div class="chi-button__content">
+//-                       <i class="chi-icon icon-chevron-left" aria-hidden="true"></i>
+//-                     </div>
+//-                   </button>
+//-                   <button class="chi-button -flat" aria-label="Page 1">1</button>
+//-                   <button class="chi-button -flat" aria-label="Page 2">2</button>
+//-                   <button class="chi-button -flat -active" aria-label="Page 3">3</button>
+//-                   <button class="chi-button -flat" aria-label="Page 4">4</button>
+//-                   <button class="chi-button -flat" aria-label="Page 5">5</button>
+//-                   <div class="chi-button -flat" aria-hidden="true" disabled>...</div>
+//-                   <button class="chi-button -flat" aria-label="Page 12">12</button>
+//-                   <button class="chi-button -icon -flat" aria-label="Next page">
+//-                     <div class="chi-button__content">
+//-                       <i class="chi-icon icon-chevron-right" aria-hidden="true"></i>
+//-                     </div>
+//-                   </button>
+//-                 </div>
+//-               </div>
+//-               <div class="chi-pagination__end">
+//-                 <div class="chi-pagination__jumper">
+//-                   <label class="chi-pagination__label" for="jumper-input-18">Go to page:</label>
+//-                   <input class="chi-input" type="text" id="jumper-input-18"/>
+//-                 </div>
+//-               </div>
+//-             </div>
+//-           </nav>
+//-         </div>
+//-       </div>
+
+h3 Portal
+.example.-mb--3
+  .-p--3
+    .chi-card.-portal
+      .chi-card__header.-sm
+        .chi-card__title Title
+      .chi-card__content.-p--0
```

Comment:
The card background-color appears to be conflicting with the cell background-color on mobile. Is the card content missing a responsive padding class or a color change?

Reply:
Card background-color comes from -chi-card -portal and it can't be overridden from data-table. So I have used responsive color utility classes to solve background-color issue.